src/layers/learning/distconv/distconv_layers.cpp

```diff
@@ -51,8 +51,8 @@ namespace distconv{
     const auto& input_dims = input.get_local_shape();
     const auto& output_dims = output.get_local_shape();
 
-    const auto& input_size = std::accumulate(input_dims.begin(), input_dims.begin()+1, 1, std::multiplies<size_t>());
```

Comment:
@szaman19 I think that this is trying to get `input_dims[0] * input_dims[1]`

Reply:
Yes, that's correct. 

We usually expect input_dims[1] == 1. The extra dimension is just needed to make the layer work nicely with distconv's channel-wise distribution. The unit test assumes that case as well.